html_layout.py

```diff
@@ -465,6 +616,18 @@ def create_appLayout(
                                 id="PUE_input",
                                 style=dict(display='none'),
                             ),
+
+                            html.Div(
+                                [
+                                    html.Div('i', className='tooltip-icon'),
+                                    html.P(
+                                        "The PUE is strictly higher than 1: 1.67 was the average worldwide value in 2019.",
```

Comment:
Well, technically it could be equal to 1, it just can't be lower. I'm not sure if this (i) really adds information about how to fill in the form?

Reply:
"The PUE accounts for the extra energy consumption of the data centre, including cooling (1.67 by default)."